# DP: Changed compile time configuration options:
# DP: 
# DP: - Set the default path to comply with Debian policy
# DP: 
# DP: - Enable System-wide .bashrc file for interactive shells.
# DP: 
# DP: - Enable System-wide .bash.logout  file for interactive shells.
# DP: 
# DP: - make non-interactive shells begun with argv[0][0] == '-'
# DP:   run the startup files when not in posix mode.
# DP: 
# DP: - try to check whether bash is being run by sshd and source
# DP:   the .bashrc if so (like the rshd behavior).
# DP: 
# DP: - don't define a default DEFAULT_MAIL_DIRECTORY, because it
# DP:   can cause a timeout on NFS mounts.

--- a/bash/config-bot.h
+++ b/bash/config-bot.h
@@ -196,4 +196,4 @@
 /******************************************************************/
 
 /* If you don't want bash to provide a default mail file to check. */
-/* #undef DEFAULT_MAIL_DIRECTORY */
+#undef DEFAULT_MAIL_DIRECTORY
--- a/bash/config-top.h
+++ b/bash/config-top.h
@@ -54,14 +54,14 @@
 /* The default value of the PATH variable. */
 #ifndef DEFAULT_PATH_VALUE
 #define DEFAULT_PATH_VALUE \
-  "/usr/gnu/bin:/usr/local/bin:/bin:/usr/bin:."
+  "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
 #endif
 
 /* The value for PATH when invoking `command -p'.  This is only used when
    the Posix.2 confstr () function, or CS_PATH define are not present. */
 #ifndef STANDARD_UTILS_PATH
 #define STANDARD_UTILS_PATH \
-  "/bin:/usr/bin:/sbin:/usr/sbin:/etc:/usr/etc"
+  "/bin:/usr/bin:/sbin:/usr/sbin"
 #endif
 
 /* Default primary and secondary prompt strings. */
@@ -75,20 +75,20 @@
 #define KSH_COMPATIBLE_SELECT
 
 /* System-wide .bashrc file for interactive shells. */
-/* #define SYS_BASHRC "/etc/bash.bashrc" */
+#define SYS_BASHRC "/etc/bash.bashrc"
 
 /* System-wide .bash_logout for login shells. */
-/* #define SYS_BASH_LOGOUT "/etc/bash.bash_logout" */
+#define SYS_BASH_LOGOUT "/etc/bash.bash_logout"
 
 /* Define this to make non-interactive shells begun with argv[0][0] == '-'
    run the startup files when not in posix mode. */
-/* #define NON_INTERACTIVE_LOGIN_SHELLS */
+#define NON_INTERACTIVE_LOGIN_SHELLS
 
 /* Define this if you want bash to try to check whether it's being run by
    sshd and source the .bashrc if so (like the rshd behavior).  This checks
    for the presence of SSH_CLIENT or SSH2_CLIENT in the initial environment,
    which can be fooled under certain not-uncommon circumstances. */
-/* #define SSH_SOURCE_BASHRC */
+#define SSH_SOURCE_BASHRC
 
 /* Define if you want the case-capitalizing operators (~[~]) and the
    `capcase' variable attribute (declare -c). */
